src/bioversions/sources/rxnorm.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+"""A getter for RxNorm."""
+
+from datetime import datetime
+
+from ..utils import Getter, VersionType, get_soup
+
+__all__ = [
+    "RxNormGetter",
+]
+
+URL = "https://www.nlm.nih.gov/research/umls/rxnorm/docs/rxnormfiles.html"
+
+
+class RxNormGetter(Getter):
+    """A getter for RxNorm."""
+
+    bioregistry_id = "rxnorm"
+    name = "RxNorm"
+    homepage_fmt = "https://download.nlm.nih.gov/umls/kss/rxnorm/RxNorm_full_{version}.zip"
+    version_type = VersionType.date
+    date_fmt = "%m%d%Y"
+
+    def get(self) -> str:
+        """Get the latest BioGRID version number."""
+        soup = get_soup(URL)
+        raw_version = soup.find("th", {"class": "current"}).contents[2]
+        raw_fmt = "%B %d, %Y"
+        dt = datetime.strptime(raw_version, raw_fmt)
+        version = datetime.strftime(dt, "%m%d%Y")
```

Comment:
why did you choose to use this format? YYYY-MM-DD is much better (for example, it's sortable)

Reply:
Agreed! I wasn't sure whether I was constrained to matching the homepage_fmt string. Is something like this possible?

Reply:
I'm a bit busy at the moment to grok this, but check this example https://github.com/biopragmatics/bioversions/blob/main/src/bioversions/sources/drugbank.py that shows where I had to put a transformation on the version before I formatted it in there